src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/markup/Text.java

```diff
@@ -47,6 +48,7 @@ public class Text extends Content {
      * @return the object
      */
     public static Text of(CharSequence content) {
+        assert checkNewlines(content);
```

Comment:
Redundant assert as private constructor also contains it.

Reply:
Fixed